### ISSUE TITLE
Annotate user name in `Creator`

### ISSUE THREAD

**Issue by bertsky** [OCR4all/LAREX#188]
When many different annotators cover a stock of documents, they are sometimes inconsistent or need specific post-processing (or just need to know which files are "theirs"). Thus it would be highly useful to make use of `/PcGts/Creator` (which is currently empty) in the result.

Tomcat has full-blown user management, so I guess it would not be too difficult to query the user name?

**Comment by maxnth** [comment]
While I agree that this would indeed be a useful feature I'm not sure that using – for example – the current tomcat user wouldn't exclude people running LAREX with jetty etc.

My suggestion would be to add a button + modal in the viewer which opens a form for manually entering all possible PAGE XML metadata. 
One could allow the user to set a temporary user name (either in `localstorage` or for a session) so that this username automatically gets added to all pages the user edits/saves while the temporary user name is set.

Would that be helpful in your use case as well?

**Comment by bertsky** [comment]
> My suggestion would be to add a button + modal in the viewer which opens a form for manually entering all possible PAGE XML metadata (Creator, notes, …).

Yes, and preferably as part of the _segmentation settings_ (`SegmentationSettingsReader` etc), so they can be easily saved/loaded.

> One could allow the user to set a temporary user name (either in `localstorage` or for a session) so that this username automatically gets added to all pages the user edits/saves while the temporary user name is set.

I don't know enough about application server programming to judge how this could/should be implemented.

> Would that be helpful in your use case as well?

Absolutely. Tomcat username was just my first thought.

**Comment by maxnth** [comment]
Modal to view and edit basic metadata (creator, comments, externalRef) was added in ef2c23385b1504119610bc5ed216310b9a5fcf7c
More complex metadata / attribute editing will get added later on

**Comment by bertsky** [comment]
It would still be great if it were possible to have _some_ metadata pre-filled for _all_ pages (like author/creator, ID/external-ref, comments)...
